app/controllers/application_controller.rb

```diff
@@ -344,7 +348,7 @@ def analytics_exception_info(exception)
   end
 
   def add_sp_cost(token)
-    Db::SpCost::AddSpCost.call(sp_session[:issuer].to_s, sp_session[:ial2] ? 2 : 1, token)
+    Db::SpCost::AddSpCost.call(sp_session[:issuer].to_s, sp_session_ial, token)
```

Comment:
As a heads up, this will include values that are not IAL 1 or 2 now like possibly `22` --- do we want to clean those back up?

Reply:
OK, cleanup complete.